### PR TITLE
Patch/Link styling

### PR DESCRIPTION
**Pull request by Quipex**
Changed links styling. Changed text for further possibility of adding TMarket link

**No hover**
image

**When hovered on**
image

### CODE REVIEW

File: RatScanner/View/MainWindow.xaml [hunk 1]
```diff
@@ -11,6 +11,18 @@
...
+      <Setter Property="TextBlock.FontSize" Value="14" />
+      <Setter Property="Foreground" Value="White" />
+      <Style.Triggers>
+        <Trigger Property="IsMouseOver" Value="True">
+          <Setter Property="Foreground" Value="#569CD6" />
+        </Trigger>
+      </Style.Triggers>
+    </Style>
+  </Window.Resources>
+
```
Comment (Blightbuster) [inline RatScanner/View/MainWindow.xaml:25]: This should probably be a global style for Hyperlinks.
Moving this to `App.xaml` will do the trick.

File: RatScanner/View/MainWindow.xaml [hunk 2]
```diff
@@ -135,10 +147,10 @@
         <Separator/>
 
         <Grid Margin="0,5">
-          <Label Grid.Column="0" Content="Wiki Link"/>
-          <TextBlock Grid.Column="1">
-            <Hyperlink NavigateUri="{Binding WikiLink}" RequestNavigate="HyperlinkRequestNavigate" TextDecorations="">
-              <Label Content="Click to open"/>
+          <Label Grid.Column="0" Content="Links" VerticalAlignment="Center"/>
```
Comment (Blightbuster) [inline RatScanner/View/MainWindow.xaml:150]: `VerticalAlignment="Center"` is redundant here

File: RatScanner/View/MainWindow.xaml [hunk 3]
```diff
@@ -135,10 +147,10 @@
...
+          <TextBlock Grid.Column="1" VerticalAlignment="Center">
+            <Hyperlink NavigateUri="{Binding WikiLink}" RequestNavigate="HyperlinkRequestNavigate">
+              <Underline>Wiki</Underline>
```
Comment (Blightbuster) [inline RatScanner/View/MainWindow.xaml:153]: This needs 5 right margin

**Review by Blightbuster — CHANGES_REQUESTED** [review]
Thanks for the PR 👍 

Id appreciate it if you make the changes. 😄

File: RatScanner/View/MainWindow.xaml [hunk 4]
```diff
@@ -11,6 +11,18 @@
         ResizeMode="CanMinimize"
         Height="400" Width="300">
 
+  <Window.Resources>
+    <Style TargetType="{x:Type Hyperlink}">
+      <Setter Property="TextBlock.FontSize" Value="14" />
+      <Setter Property="Foreground" Value="White" />
```
Comment (Blightbuster) [inline RatScanner/View/MainWindow.xaml:17]: Color should be `#EEEEEE` instead of `White`.

Comment (Blightbuster) [inline RatScanner/View/MainWindow.xaml:17]: Missing `<Setter Property="FontFamily" Value="Roboto Medium"/>` for styling like other text or is this a deliberate choice?

File: RatScanner/View/MainWindow.xaml (hunk 1 above)
Comment (Quipex) [reply RatScanner/View/MainWindow.xaml:25]: Right, my bad, haven't seen App.xaml

Comment (Quipex) [reply RatScanner/View/MainWindow.xaml]: Done

File: RatScanner/View/MainWindow.xaml (hunk 3 above)
Comment (Quipex) [reply RatScanner/View/MainWindow.xaml:141]: Done

File: RatScanner/View/MainWindow.xaml (hunk 2 above)
Comment (Quipex) [reply RatScanner/View/MainWindow.xaml]: Removed

File: RatScanner/View/MainWindow.xaml (hunk 4 above)
Comment (Quipex) [reply RatScanner/View/MainWindow.xaml]: Done

Comment (Quipex) [reply RatScanner/View/MainWindow.xaml]: Added missing props

**Review by Blightbuster — APPROVED** [review]

**Pull request merged**